### ISSUE TITLE
Some Map  layers that use to work fine no longer working

### ISSUE THREAD

**Issue by Moshe81** [IsraelHikingMap/Site#1114]
## Bug
On the web site I have around 10 transperent layers.
some of them are for strava.
I notice that the strava layer is no longer working - https://anygis.ru/api/v1/Tracks_Strava_RunAndRide/{x}/{y}/{z}]

So I thioghot its due to the server it use.
but I notice other layers on other server also not working:
Open-Signal - https://israelhiking.osm.org.il/proxy/https%3A%2F%2Ftiles-prod.opensignal.com%2F%3Fzoom%3D{z}%26x%3D{x}%26y%3D{y}

CellID - http://opencellid.org/images/maps/opencellid/{z}/{x}/{y}.png
CellMapper - https://israelhiking.osm.org.il/proxy/https%3A%2F%2Fapi.cellmapper.net%2Fv6%2FgetTile%3FMCC%3D425%26MNC%3D2%26RAT%3DLTE%26z%3D{z}%26x%3D{x}%26y%3D{y}%26band%3D0

OTher layers like google maps or Waze are working ok.
So I think there is an issue that only impact some type of map layers and not all of them.
Issue started only during this week.

## What I expect to happen
Expect the layers to show that data on top of the main map

## What really happened
There is no data retrive for some map layers

## My environment
* Operating System and version:
Windows 10 RS5 
* Web Browser and version:
Chrom
## Thing I think the developers should know, images, links etc.

That it used to work ok untill a week ago

Thanks
MOshe

**Comment by HarelM** [comment]
Thanks for reporting this issue.
This might be a bug after a recent server update but our proxy is not intended for this usage.
Using Strava layer outside of Strava site is forbidden as I have mentioned in my posts on Facebook so we do not support it - we encourage you not to use this layer.
Other layers do not support either CORS or HTTPS or both so you should contact their owners in order to fix this - if the owner of the layer does not change it to CORS and HTTPS it means he does not want us to use these layers and it makes them forbidden as well.
We can't support infringing copyright laws. Sorry...

**Comment by Moshe81** [comment]
Thanks for the fast respond.
Feeling bad about losing this great layers but I totally understand the legal point here.

Thanks
Moshe

**Comment by Moshe81** [comment]
What does it mean CORES - why does this map not working - not HTTPS and no using PROXY http://opencellid.org/images/maps/opencellid/{z}/{x}/{y}.png 

looks like a naive url :-)

**Comment by HarelM** [comment]
CORS is the initials of Cross Origin Resource Sharing. In plain English it means that my server allows other site to use my resources (Images, files, etc).
The above URL is HTTP and **not** HTTPS. The site is secured and only shows resources from secured sites (HTTPS is a security layer over HTTP). This is a chrome security feature, Firefox for example do not enforce it. Having said that, I do not recommend using Firefox...

**Comment by Moshe81** [comment]
mmmm.... so I tired Firefox ...
Not working also with Firefox.
In other words because IHM mape is secure (HTTPS) Chrome will allow it to bring data only from Secure websites...and since they are not HTTPS sites it will be blocked - I think I got it.
But it used to work with the old server - so the old server was not secure ?
by the way I think it also introduce some desegregation in performance but its hard to say as I dont have measurements.
For my mapping activities I think I can bare using Firefox for having this extra layer (not for the bak activities)
